packages/transport/src/modules/accounts/IdentityController.ts

```diff
@@ -3,9 +3,13 @@ import { CoreBuffer, CryptoSignature, CryptoSignaturePrivateKey, CryptoSignature
 import { ControllerName, CoreAddress, CoreCrypto, CoreErrors, TransportController } from "../../core";
 import { AccountController } from "../accounts/AccountController";
 import { DeviceSecretType } from "../devices/DeviceSecretController";
-import { Identity, Realm } from "./data/Identity";
+import { SynchronizedCollection } from "../sync/SynchronizedCollection";
+import { Identity } from "./data/Identity";
+import { Realm } from "./data/Realm";
```

Comment:
Why was Realm moved?

Reply:
it was to prevent cyclic dependencies when we had a dependency from Identity to IdentiyDeletionProcess